.travis_build_trigger

```diff
@@ -0,0 +1 @@
+this file is only used to trigger a travis build
```

Comment:
I will not accept this in a pull request about locale

Reply:
Agreee. that should have gone to my fork instead. I will sort out Roman's and my contibutions and submit a separate PR whan you are ready.